src/main/java/at/hannibal2/skyhanni/features/garden/composter/ComposterOverlay.kt

```diff
@@ -414,8 +417,29 @@ class ComposterOverlay {
             list.add(Renderable.link("$name§r §8x${itemsNeeded.addSeparators()} §7(§6$format§7)") {
                 onClick(internalName)
                 if (LorenzUtils.isControlKeyDown()) {
-                    inInventory = false
-                    BazaarApi.searchForBazaarItem(itemName, itemsNeeded.toInt())
+                    if(lastAttemptTime.passedSince()>0.5.seconds){
+                        lastAttemptTime = SimpleTimeMark.now()
+                        val amountInSacks = fetchSackItem(internalName.asInternalName())?.amount
+                        if (itemsNeeded.toInt() != 0 && itemName.removeColor() != "Biofuel") {
+                            if (config.composterOverlayGetType == 0) {
+                                inInventory = false
+                                BazaarApi.searchForBazaarItem(itemName, itemsNeeded.toInt())
+                            } else if (amountInSacks == 0) {
+                                SoundUtils.createSound("mob.endermen.portal", 0F).playSound()
+                                chat("§e[SkyHanni] No $itemName §r§efound in sacks. Opening bazaar.")
```

Comment:
i think if the fetched item's status is -1 it should tell them to open their sacks to update the data as there is none saved